lib/tests/test_commit_builder.rs

```diff
@@ -14,12 +14,12 @@
 
```

Comment:
nit: stale subject line in commit description?
> tests: add `create_tree()` returning `MergedTree` and use where easy

Reply:
Oops, yes. Removed.